packages/embark-pipeline/src/index.js

```diff
@@ -179,7 +178,7 @@ class Pipeline {
         // assuming we got here because an asset was changed, let's check our webpack config
         // to see if the changed asset requires webpack to run
         if (!(modifiedAssets && modifiedAssets.length)) return next(null, false);
-        const configReader = new WebpackConfigReader({webpackConfigName: self.webpackConfigName});
+        const configReader = new WebpackConfigReader({webpackConfigName: self.webpackConfigName, fs: self.embark.fs});
```

Comment:
I know you were doing / did some work in another PR related to the injected fs, so I'm curious if `fs: self.embark.fs` is still needed.

Reply:
You're absolutely right, this is not needed anymore.

Good catch!